src/Config/ProcessableResource.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusImagePlugin\Config;
+
+final class ProcessableResource
+{
+    private string $resource;
```

Comment:
I would love some comments on these properties to explain what they hold. It's only to heighten the developer experience ;)